src/polyglot-notebooks-vscode-common/src/metadataUtilities.ts

```diff
@@ -281,6 +279,8 @@ export function getKernelspecMetadataFromNotebookDocumentMetadata(notebookDocume
 export function createNewIpynbMetadataWithNotebookDocumentMetadata(existingMetadata: { [key: string]: any }, notebookDocumentMetadata: NotebookDocumentMetadata): { [key: string]: any } {
     const resultMetadata: { [key: string]: any } = { ...existingMetadata };
 
+    // FIX: "custom" actually means just the bucket of things we don't persist, and therefore should not cause the document dirty state to be set
```

Comment:
minor: Is this something you intended to address before merging this PR or is this just todo placeholder for a future fix?

Reply:
These are notes and placeholders for future fixes.